autospec/buildreq.py

```diff
@@ -19,16 +19,15 @@
 # Deduce and manage build requirements
 #
 
+import ast
 import os
 import re
-import ast
-
-import toml
+import subprocess
 
 import buildpattern
-import util
 import config
-import subprocess
+import toml
```

Comment:
flake8 was yelling if I tried to split up third party from autospec because it seemed to detect autospec as 3rd party. It also detected test as stdlib instead of part of autospec now that I think of it.

Reply:
Okay, per our offline discussion, it looks like what I am proposing is not possible. The other details you mention complicate the situation even more :-(